pkg/assembler/graphql/schema/certifyVuln.graphql

```diff
@@ -129,7 +137,16 @@ VulnerabilityMetaDataInput represents the input for certifying vulnerability
 scans in mutations.
 """
 input VulnerabilityMetaDataInput {
-  timeScanned: Time!
+  "cvss 2.0 score, defaults to empty string"
+  cvss2Score: Float!
+  "cvss 3.0 score, defaults to empty string"
+  cvss3Score: Float!
+  "additional information, defaults to empty string"
+  additionalInformation: String!
+  "links pointing to other resources, defaults to empty list"
+  links: [String!]!
+  ollector: String!
+  timeStamp: Time!
```

Comment:
will also have to add a filter called `VulnerabilityMetaDataSpec` (for consistency, for now, prob need to rename it to filter in the future).

```
type VulnerabilityMetadataSpec {
  id: ID
  vulnerability: Vulnerability
  cvss2Score: Float
  cvss3Score: Float
  timeStamp: Time
  additionalInformation: String
  links: [String!] = []
  origin: String
  collector: String
}
```

Do we need to add in a `matchOnlyEmptyLinks`? Seems like in this case that would not be needed. @mihaimaruseac?

Finally will also have to add the query and mutation similar to the `certifyVuln` but update fr the `Vulnerability Metadata`:

```
extend type Query {
  "Returns all vulnerability certifications matching the input filter."
  CertifyVuln(certifyVulnSpec: CertifyVulnSpec): [CertifyVuln!]!
}

extend type Mutation {
  "Adds a certification that a package has been scanned for vulnerabilities."
  ingestVulnerability(pkg: PkgInputSpec!, vulnerability: VulnerabilityInput!, certifyVuln: VulnerabilityMetaDataInput!): CertifyVuln!
}
```

Reply:
I don't think we need the `machOnlyEmptyLinks` here, since these are not actual identifiers (we can have the same set of links on two separate nodes)